src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -16,19 +17,17 @@ namespace NuGet.PackageManagement.UI
     /// <summary>
     /// Interaction logic for PackageReadmeControl.xaml
     /// </summary>
-    public partial class PackageReadmeControl : UserControl, IDisposable
+    public partial class PackageReadmeControl : UserControl
     {
 #pragma warning disable CS0618 // Type or member is obsolete
-        private IMarkdownPreview _markdownPreview;
+        public IMarkdownPreview markdownPreview;
```

Comment:
Can we make this a property instead? It's best not to have public instance variables.